internal/typeinfo/slice.go

```diff
@@ -0,0 +1,28 @@
+package typeinfo
```

Comment:
How is the organization working? I can see a lot of common interfaces in `member.go`  for example `Input` and `Output` and then slices are going on a different file altogether.

Reply:
Ah, I didn't think when I created the file. I've changed member.go to valuelocator.go and moved slice.go in there.